modules/tensor_mechanics/doc/content/documentation/systems/BCs/CoupledPressureBC.md

```diff
@@ -0,0 +1,17 @@
+# Coupled Pressure BC
+
+## Description
+
+The boundary condition `CoupledPressureBC` applies a force computed in a variable to a mesh boundary.
+A `component` of the normal vector to the mesh surface (0, 1, or 2 corresponding to the $\hat{x}$, $\hat{y}$, and $\hat{z}$ vector components) is used to determine the direction in which to apply the traction.
+The boundary condition is always applied to the displaced mesh.
+
+The magnitude of the `Pressure` boundary condition has to be computed as an variable and coupled into this class.
```

Comment:
Add an equation to demonstrate the calculation that your class computes here please. I agree that it's straightforward, but sometimes an (equation) picture is worth a thousand words, especially to newer users trying to find their footing

Reply:
This class will be used in 95% of cases in a multi-app scenario. The pressure will be computed for example by a flow code, like RELAP-7. There we have 2 pressures, we compute a mixture pressure and that is transferred into aux-variable. Then this aux is used by this BC.  Other case is the SAM code which computes pressure in a totally different way (they actually solve for pressure). But the use case is the same. It is transferred into an aux and used by this BC.  That's why I did not put more specific details. But I could describe this here...

Reply:
Ah, well, in that case, let's leave it as it is and add a short note about the intended use being for multi-apps. The details of the actual use case would probably too confusing for the brand new user